packages/nx/src/native/native_hasher.rs

```diff
@@ -28,25 +28,25 @@ fn hash_file(file: String) -> Option<FileData> {
 
 #[napi]
 fn hash_files(workspace_root: String) -> HashMap<String, String> {
-    let mut walker = WalkBuilder::new(&workspace_root);
     let workspace_root = Path::new(&workspace_root);
-    walker.add_ignore(workspace_root.join(".nxignore"));
```

Comment:
For some reason, the `add_ignore` here only used the first item in the ignore file 🤨